go.mod

```diff
@@ -172,6 +173,7 @@ require (
 	github.com/tklauser/numcpus v0.6.1 // indirect
 	github.com/vcaesar/cedar v0.20.1 // indirect
 	github.com/vmihailenco/tagparser/v2 v2.0.0 // indirect
+	github.com/weaviate/fgprof v0.0.0-20240724124431-aab99fd9c876 // indirect
```

Comment:
we have our own fork of `fgprof`?

Reply:
yes, see PR description 😅 

> This is currently using a fork that adds the functionality of ignoring some functions in the profiling - otherwise our tickers would crowd everything else out. I will try to upstream it in the future, but do not want to wait until upstream accepts it and makes a new release